static/js/components/followup_request/FollowupRequestForm.jsx

```diff
@@ -287,14 +287,21 @@ const FollowupRequestForm = ({
     if (
       !isSomeActiveRangeOrNoRange(
         allocationLookUp[selectedAllocationId].validity_ranges,
-        new Date(formData.start_date),
+        new Date(formData.start_date || Date.now()),
```

Comment:
do you need to put Date.now() IN Date()??? Can't it just be outside it. Also isn't it an issue that formData.start_date is likely UTC time (or at least some string that's agnostic to timezones), whereas Date.now() is likely local time?

Reply:
Got it